custom/plugins/VirtuaFeaturedProducts/Resources/views/frontend/detail/index.tpl

```diff
@@ -5,4 +5,5 @@
     {if $sArticle.is_featured == 1}
```

Comment:
Widgeti displayed in invalid place:
> display it on PDP under product base info.

See task description in jira, there is a screenshot with correct place